.gitignore

```diff
@@ -3,4 +3,5 @@
 /cargo/
 /build/
 /.flatpak-builder/
-/src/config.rs
\ No newline at end of file
+/src/config.rs
+/subprojects/blueprint-compiler
```

Comment:
did you end up adding the subproject in the end? I don't see it in the tree (the wrap file)

(not that we absolutely need to)